src/app/workspace/organization/organization-settings.component.html

```diff
@@ -1,98 +1,103 @@
-<div class="container-fluid" *ngIf="organization">
-  <div class="row">
-    <div class="col"></div>
-    <div class="whole col-12 col-lg-11 col-xl-10" [style.opacity]="initialized() ? '1' : '0'">
-      <div class="d-flex flex-row justify-content-start">
-        <a class="fas fa-chevron-left bigger mr-2" [routerLink]="['/workspace']"></a>
-        <i class="color-class fa-fw {{ organization.icon }} organization-icon clickable"
-           [style.color]="organization.color"
-           data-toggle="dropdown"></i>
-        <div *ngIf="initialized()" class="dropdown-menu">
-          <color-picker class="dropdown-item" [(color)]="organization.color"
-                        (selectedChange)="updateOrganization()"></color-picker>
-          <div class="dropdown-divider"></div>
-          <icon-picker class="dropdown-item" [(icon)]="organization.icon"
-                       (selectedChange)="updateOrganization()"></icon-picker>
-        </div>
-
-        <input class="color-class name edit-border" [(ngModel)]="organization.name" (blur)="name.valid && updateOrganizationName()"
-               name="name"
-               [style.color]="organization.color" i18n-title="@@header.organization.name"
-               title="Edit name of the organization"
-               #name="ngModel" required minlength="3" maxlength="24">
+<div class="col-12 col-lg-11 col-xl-10 m-auto" *ngIf="organization">
```

Comment:
This component is just huge. We should try to split it into the smaller ones in the future.

Reply:
Do you think that making `resource-header` is good idea? I was thinking about that, but there are some differences so I am not sure about that.. But of course I can split this component into the smaller ones..

Reply:
I have not really analyzed it into the depth. If we create two components that are similar we can then work further and extract them into the new abstract one.